rxbindings/src/test/java/com/amplifyframework/rx/RxAuthBindingTest.java

```diff
@@ -137,7 +137,7 @@ public void testSignUpFails() throws InterruptedException {
         AuthSignUpOptions options = AuthSignUpOptions.builder().build();
 
         // Arrange a callback on the failure consumer
-        AuthException failure = new AuthException("Sign up", "has failed");
+        AuthException failure = new AuthException("Sign up", "has failed", null);
```

Comment:
Since cause has default value of null, do we need to specify null explicitly?

Reply:
Yes, this is java and we don't have jvm overloads